custom_components/myskoda/coordinator.py

```diff
@@ -64,10 +64,16 @@ def __init__(
 Operations = OrderedDict[str, EventOperation]
 
 
+@dataclass
+class Config:
+    auxiliary_heater_duration: float | None = None
```

Comment:
Can you explain how this config works? I can see if unset in `self.data` we'll create a default `Config` object (which will have `auxiliary_heater_duration=None`), but where/how do set `auxiliary_heater_duration` to something else?

Reply:
heater duration is used for cars with `AUXILIARY_HEATING` but without `AUXILIARY_HEATING_TEMPERATURE_SETTING`.. Its used to defined for how long the auxiliary heater should be running before its automatically switched off. Because it is not possible to set it via own API method like target temperature I have created own (dummy) NumberEntity sensor which holds a value defined by user. This number is send in `start_auxiliary_heating`.. So the number is set from `AuxiliaryHeaterDuration` class (number.py) and used by `AuxiliaryHeater` (climate.py)